content/en/v3/admin/guides/k8s-upgrades/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Kubernetes upgrades
+linktitle: Kubernetes upgrade
+type: docs
+description: How to upgrade the version of Kubernetes in your cluster.
+weight: 92
+---
+ 
+Upgrading your Kubernetes cluster is one of the tasks you must take on as an operator of Jenkins X. Kubernetes comes out with [new releases](https://kubernetes.io/releases/) regularly, and will only maintain releases for the three latest versions. Meaning if something breaks, or there is a security issue on an older version, Kubernetes will not fix it, but ask you to upgrade.  All the major cloud providers only support a given set of latest versions. 
+
+Jenkins X aims to support the same set of versions supported by the major cloud providers, Google, Amazon and Azure. These versions are described in the links below, ranging from version 1.20 to version 1.24.
+- [Amazon](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+- [Azure](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar)
+- [Google](https://cloud.google.com/kubernetes-engine/docs/release-notes)
```

Comment:
I would say something about recommending to follow your providers recommended guidelines.

Reply:
Yes, and in practice, we only support 1.20 and 1.21 ATM.

Reply:
Also, I think I mention that you should follow the provider guidelines in the section below